src/Symfony/Bundle/FrameworkBundle/Resources/config/object_mapper.php

```diff
@@ -29,5 +30,15 @@
                 tagged_locator('object_mapper.condition_callable'),
             ])
         ->alias(ObjectMapperInterface::class, 'object_mapper')
+
+        ->set('object_mapper.cached', CachedObjectMapper::class)
+            ->args([
+                param('.object_mapper.cache_dir'),
+                service('object_mapper.metadata_factory'),
+                service('property_accessor')->ignoreOnInvalid(),
+                tagged_locator('object_mapper.transform_callable'),
+                tagged_locator('object_mapper.condition_callable'),
+            ])
+        ->alias(ObjectMapperInterface::class, 'object_mapper')
```

Comment:
I suppose this alias should have been

```suggestion
        ->alias(ObjectMapperInterface::class, 'object_mapper.cached')
```

BUT, I'd create this alias in the FrameworkExtension and only when not in debug mode
also, we should alias object_mapper to object_mapper.cached so that one could reliably use object_mapper in service references